api/src/main/java/org/openmrs/util/databasechange/ConceptValidatorChangeSet.java

```diff
@@ -139,10 +139,10 @@ private void validateAndCleanUpConcepts(JdbcConnection connection) {
 				Map<String, List<ConceptName>> nameDuplicateConceptNamesMap = new HashMap<String, List<ConceptName>>();
 				
 				//for each name in the locale
-				for (ConceptName nameInLocale : localeConceptNamesMap.get(conceptNameLocale)) {
-					if (StringUtils.isBlank(nameInLocale.getName())) {
-						updateWarnings.add("ConceptName with id " + nameInLocale.getConceptNameId() + " ("
-						        + nameInLocale.getName() + ") is null, white space character or empty string");
+				for (Entry<String, List<ConceptName>> entry : localeConceptNamesMap.entrySet()) {
```

Comment:
You have just changed the whole buisness logic. We must only iterate over names in the specific locale. Please revert.